src/ethereum_test_forks/forks/forks.py

```diff
@@ -989,13 +989,13 @@ def pre_allocation_blockchain(cls) -> Mapping:
         type tests.
         """
         new_allocation = {
-            Address(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE): {
-                "nonce": 1,
-                "code": (
-                    "0x60203611603157600143035f35116029575f35612000014311602957612000"
-                    "5f3506545f5260205ff35b5f5f5260205ff35b5f5ffd00"
-                ),
-            }
+            # Address(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE): {
+            #     "nonce": 1,
+            #     "code": (
+            #         "0x60203611603157600143035f35116029575f35612000014311602957612000"
+            #         "5f3506545f5260205ff35b5f5f5260205ff35b5f5ffd00"
+            #     ),
+            # }
```

Comment:
@spencer-tb, this is a quick hack to disable the automatic inclusion of the EIP-2935 system contract. Ideally, I want this feature to avoid having this extra address in the genesis state since we want full predictability on which accounts exist for the tree conversion tests.

I think once we talked about an extra flag to do this more cleanly.